src/test/ui/rfc-2632-const-trait-impl/staged-api.rs

```diff
@@ -16,12 +14,11 @@ use staged_api::*;
 pub struct Stable;
 
 #[stable(feature = "rust1", since = "1.0.0")]
-#[cfg_attr(staged, rustc_const_stable(feature = "rust1", since = "1.0.0"))]
-// ^ should trigger error with or without the attribute
+#[cfg_attr(stable, rustc_const_stable(feature = "rust1", since = "1.0.0"))]
 impl const MyTrait for Stable {
-    fn func() { //~ ERROR trait methods cannot be stable const fn
-
-    }
+    //[stable]~^ ERROR trait implementations cannot be const stable yet
+    //[unstable]~^^ ERROR implementation has missing const stability attribute
```

Comment:
This ensures that `impl const Trait` will require a const stability attribute once it's stabilized, just like everything else (#90998).